### PR TITLE
Importing content from deprecated telemetry-framework repo

### PR DESCRIPTION
**Pull request by csibbitt**
* From an up to date master of both
```
$ cp -a ../telemetry-framework/deploy/ .
$ rm deploy/configs/es-workaround.bash  # We don't need this anymore
$ cp -a ../telemetry-framework/tests/ .
$ git add -A
```
* Merged important content from README.md

History before this point may be available at
https://github.com/redhat-service-assurance/telemetry-framework

Also, see this PR: https://github.com/redhat-service-assurance/telemetry-framework/pull/117

More detail on Trello 898

### CODE REVIEW

**Comment by csibbitt** [comment]
[Move testing (and quickstart) artifacts from telemetry-framework to SAO repo](https://trello.com/c/ZpAvTdgM/898-move-testing-and-quickstart-artifacts-from-telemetry-framework-to-sao-repo)

File: deploy/quickstart.sh [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+---
+
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: serviceassurance-operator-alpha-redhat-service-assurance-operators-openshift-marketplace
+  namespace: ${SAF_PROJECT}
+spec:
+  channel: alpha
```
Comment (mrunge) [inline deploy/quickstart.sh]: beta

File: deploy/quickstart.sh [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  installPlanApproval: Automatic
+  name: serviceassurance-operator
+  source: redhat-service-assurance-operators
+  sourceNamespace: openshift-marketplace
+  startingCSV: service-assurance-operator.v0.1.0
```
Comment (mrunge) [inline deploy/quickstart.sh]: startingCSV: service-assurance-operator.v0.1.1

File: deploy/quickstart.sh [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+spec:
+  targetNamespaces:
+  -  ${SAF_PROJECT}
+
+---
+
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: serviceassurance-operator-alpha-redhat-service-assurance-operators-openshift-marketplace
```
Comment (mrunge) [inline deploy/quickstart.sh]: name: serviceassurance-operator

File: deploy/quickstart.sh (hunk 1 above)
Comment (leifmadsen) [reply deploy/quickstart.sh]: All of these should be available on merge or rebase from master.

Comment (mrunge) [reply deploy/quickstart.sh]: I checked before, see https://github.com/redhat-service-assurance/telemetry-framework/blob/master/deploy/quickstart.sh#L54

Comment (csibbitt) [reply deploy/quickstart.sh]: This is a great argument for merging the repos! :) Updating now.

Comment (csibbitt) [reply deploy/quickstart.sh]: Now with the quickstart in this repo, I might be able to eliminate the duplication and have it re-use artifacts from the source like the old quickstart did.

File: deploy/quickstart.sh (hunk 3 above)
Comment (csibbitt) [reply deploy/quickstart.sh]: IIRC the name that's in there matches the one on the subscription that is created when you install via the console UI. What leads you to suggest this change?

Comment (mrunge) [reply deploy/quickstart.sh]: The output I got from oc describe ... 
Let's go with this, if you all tested it, then I may have done a mistake in my test.

**Pull request merged**